scripts-endpoint.md

```diff
@@ -0,0 +1,169 @@
+# Admin Scripts Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+DSpace (6) has admin functionality to import and export items in CSV and ZIP format, to start a collection harvest run, to run or schedule curation tasks. Each of these functionalities also map on a DSpace CLI script. While we could implement each of these operations as a separate endpoint, this contract tries to describe a generic endpoint that allows administrators to explore, run or schedule DSpace CLI scripts from the REST API.
+
+## Scripts Endpoint
+**GET /api/admin/scripts**
+
+This endpoint will list all (REST supported) scripts defined in `dspace/config/launcher.xml`. The script entries are embedded with a name, description and a self link. By "REST supported" we mean all scripts that have been updated to allow invocations from the REST API.
+
+The JSON response document is as follows
+```
+{
+  "page": {
+      	"size": 5,
+      	"totalElements": 14,
+      	"totalPages": 3,
+      	"number": 0
+  },
+  "sort" : {
+    "by" : "name",
+    "order" : "asc"
+  },
+  "_embedded" : {
+    "scripts" [
+      {
+        "name" : "import",
+        "description" : "Import items into DSpace"
+        "_links" : {
+          "self" : {
+            "href" : "/api/admin/scripts/import"
+          }
+        }
+      },
+      {
+        "name" : "metadata-import",
+        "description" : "Import metadata after batch editing"
+        "_links" : {
+          "self" : {
+            "href" : "/api/admin/scripts/metadata-import"
+          }
+        }
+      },
+      ...
+    ]
+  }
+```
+
+## Script Name Endpoint
+
+### Script Details
+**GET /api/admin/scripts/<:script-name>**
```

Comment:
I don't like the admin category, it seems related to an authz concept that I won't enforce. These scripts are essentially configuration so why don't use the already defined config category?

Reply:
I'm OK with using "config" here.